index.rst

```diff
@@ -6,65 +6,110 @@
-Because of the heat generated by such a lamp, before sending the light into the monochromator, the light travels through a water source which absorbs much of the infrared wavelengths. This water heats up, so the white light source must be operated with a chiller, discussed below. The chiller should keep the water cool enough, but if there is ever a situation where there is too much heat being sent into the monochromator, there is a thermal fuse glued attached to the entrance to the monochromator. When it hits a certain temperature it will trigger the emergency stop. On the mountain, there are also temperatures that will freeze the water in the cell. In the event of this occurance, there are heaters that are attached to the cell which will be turned on at a certain temperature, as determined by the operator. The heaters are controlled by an `Omron temperature Controller <http://products.omron.us/item/in-panel-controllers/e5dc/e5dc-qx2dsm-002>`__
+Because of the heat generated by such a lamp, before sending the light into the monochromator, the light travels through a water cell which absorbs much of the infrared radiation. This water heats up, so the white light source must be operated with a chiller, discussed below. The chiller should keep the water cool enough, but if there is ever a situation where there is too much heat being sent into the monochromator, there is a thermal fuse attached to the entrance to the monochromator. When it hits a certain temperature it will trigger the emergency stop. The WLS will also experience freezing temperatures on the summit. To avoid freezing the water in the cell, there are heaters attached to the cell which will be turned on at a certain temperature, as determined by the operator. The heaters are controlled by an `Omron temperature Controller <http://products.omron.us/item/in-panel-controllers/e5dc/e5dc-qx2dsm-002>`__
 
-The thermal fuses are set to turn off the power at 50C (TBC) and the heaters will turn on when the cell reaches 10C (TBC).
+.. note:: The bulb for the WLS is a Mercury-Xenon arc lamp. In the rare case that a mercury bulb explodes and the mercury is released from its envelope, it is recommended that that all personnel should leave the immediate area at once, so that no mercury vapor is inhaled. The area should be ventilated for a minimum of 30 minutes. When the lamp housing has cooled any mercury residue should be picked up with a special adsorptive agent. Compact arc bulbs contain a highly pressurized gas, and present an explosion hazard even when cold. If the lamp goes out, there is a spare bulb available. Please follow the instructions carefully. 
+
+The thermal fuses are set to turn off the power at 85C and the heaters will turn on when the cell reaches 5C, warming up the cell to 26C. The temperatures for the heaters are adjustable within the electronics cabinet. In order to change the temperature setting for the thermal fuse, a new fuse will have to be purchased and installed manually.
```

Comment:
I'm confused...
so when the cell hits 5C, the heaters turn on and heat it up to 26??? We just need to maintain it around ~5, so why go so hot?

Reply:
The chiller is running at 20 or 25 C. I just assumed I would warm it back up to where the chiller is warming. I could certainly be turned down.